helm/blueapi/values.yaml

```diff
@@ -68,6 +68,10 @@ affinity: {}
 
 #existingSecret: see templates/secret.yaml
 
+scratch:
+  {}
+  # hostPath: /dls_sw/p38/software/
```

Comment:
Intend to leave this in as an example unless anyone can think of a better one

Reply:
Something non-diamond specific may be useful in the long game, e.g. where we deploy such software at a non-diamond location, or for third-party users (in the event that this is useful for others)

`p38` requires some rather internal knowledge to make sense of

Reply:
Also, if this really is a scratch area, perhaps 

```
/usr/local/blueapi-software-scratch
```

would be a more explicit example (appreciate that you make assumptions in the kubernetes environment about this being available on any host, but that can be arranged)

Reply:
Good idea, chagned to that